examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p><code>start</code> indicates the fraction of the input series _after which_ the scorers should be trained on. Furthermore I wouldn't say it's best if it's small, but rather explain the tradeoff: small means more data but at the risk of using older parts of the series, whereas large means a smaller but more recent sub-series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>